great-idea/index.html

```diff
@@ -14,10 +14,77 @@
   <![endif]-->
 </head>
 
-  <body>
-
-    <!-- Copy and paste your HTML from the first UI project here -->
-      
-    
+  <body id="index">
+    <div class="container">
```

Comment:
it would probably be much safer just to put the id on the container div instead
the body tag does a lot of stuff behind the scenes so i would just avoid adding any properties to it.
a tag can have both an id and a class

Reply:
That makes sense. I'll be sure to do that from now on. Just pushed a commit with those changes.